plugin/storage/cassandra/factory.go

```diff
@@ -131,6 +137,262 @@ func (f *Factory) configureFromOptions(o *Options) {
 	}
 }
 
+const (
+	MODE               = `MODE`
+	DATACENTER         = `DATACENTER`
+	TRACE_TTL          = `TRACE_TTL`
+	DEPENDENCIES_TTL   = `DEPENDENCIES_TTL`
+	KEYSPACE           = `KEYSPACE`
+	REPLICATION_FACTOR = `REPLICATION_FACTOR`
+	VERSION            = `VERSION`
+	COMPACTION_WINDOW  = `COMPACTION_WINDOW`
+)
```

Comment:
The functionality you're adding is only needed in jaeger-v2, which uses YAML config files. So there will be no env variables - you can simply use the struct below. But you should add `mapstructure` tags to the fields, as well as validation instructions (example: cmd/jaeger/internal/extension/jaegerquery/config.go)

Reply:
Done. Thanks